src/components/tutorial/TutorialNav.astro

```diff
@@ -68,6 +68,16 @@ const isCurrentUnit = (unit: typeof units[number]) =>
 			))
 		}
 	</TabbedContent>
+
+	<div class="feedback">
+		<a href="javascript:;" 
+			data-feedback-fish
+			data-feedback-fish-path={currentUrl}
+			>
+			Give us feedback
+		</a>
```

Comment:
We should probably use a button here as the functionality is not really that of a link:

```suggestion
		<button data-feedback-fish data-feedback-fish-path={currentUrl}>
			Give us feedback
		</button>
```

Reply:
Agreed with Chris on both counts. I like a button outside of the nav.

Excited to try this!